optimum/exporters/onnx/base.py

```diff
@@ -148,7 +148,12 @@ class OnnxConfig(ExportConfig, ABC):
             }
         ),
         "semantic-segmentation": OrderedDict({"logits": {0: "batch_size", 1: "num_labels", 2: "height", 3: "width"}}),
-        "seq2seq-lm": OrderedDict({"logits": {0: "batch_size", 1: "decoder_sequence_length"}}),
+        "seq2seq-lm": OrderedDict(
+            {
+                "logits": {0: "batch_size", 1: "decoder_sequence_length"},
+                "encoder_last_hidden_state": {0: "batch_size", 1: "encoder_sequence_length"},
```

Comment:
The issue could be in general to any encoder-decoder model. Could you also check the `speech2text` models also?

Reply:
The tests pass.

Reply:
Ok lets look into it after the fix